data/json/mutations/mutations.json

```diff
@@ -8494,7 +8497,7 @@
     "points": 0,
     "vitamin_cost": 210,
     "visibility": 8,
-    "ugliness": 5,
+    "ugliness": 2,
```

Comment:
Two tentacles is still two limbs as a human would have. Four limbs in their place would look even stranger.